### ISSUE TITLE
Phylo.draw_graphviz() deprecated but still in tutorial

### ISSUE THREAD

**Issue by rwst** [biopython/biopython#1970]
BioPython current master on:
```
~/biopython/Doc/examples> ipython3 --matplotlib
Python 3.7.2 (default, Dec 30 2018, 16:18:15) [GCC]
Type 'copyright', 'credits' or 'license' for more information
IPython 6.4.0 -- An enhanced Interactive Python. Type '?' for help.
Using matplotlib backend: TkAgg

In [1]: from Bio import Phylo

In [2]: tree = Phylo.read("simple.dnd", "newick")

In [3]: Phylo.draw_graphviz(tree)
/usr/lib64/python3.7/site-packages/Bio/Phylo/_utils.py:142: BiopythonDeprecationWarning: draw_graphviz is deprecated; use Bio.Phylo.draw instead
  BiopythonDeprecationWarning)
```
But several (fanciful tree) examples in the tutorial use `draw_graphviz()`.

Also, not having graphviz I get an error:
```
During handling of the above exception, another exception occurred:

MissingPythonDependencyError              Traceback (most recent call last)
<ipython-input-3-1f5bc9d47bfa> in <module>()
----> 1 Phylo.draw_graphviz(tree)

/usr/lib64/python3.7/site-packages/Bio/Phylo/_utils.py in draw_graphviz(tree, label_func, prog, args, node_color, **kwargs)
    172     except ImportError:
    173         raise MissingPythonDependencyError(
--> 174             "Install PyGraphviz or pydot if you want to use draw_graphviz.")
    175 
    176     def get_label_mapping(G, selection):

MissingPythonDependencyError: Install PyGraphviz or pydot if you want to use draw_graphviz.
```
But the suggested pydot installation will still fail with the same message.

As next step I would remove the resp. examples from the tutorial. Agree?

**Comment by peterjc** [comment]
Probably best to convert the examples to using ``Bio.Phylo.draw`` instead or removing them, but yes, something needs updating on the Tutorial.

As to the pydot issue, as  I recall that was a pain on Python 3 (there was an updated fork, not sure if the main project is up to date now  - would have to check). Do you want to look at that more, or just ignore GraphViz for your own needs?

**Comment by rwst** [comment]
> ... convert the examples to using Bio.Phylo.draw ...

This will not work as you won't get vertical cladogram or free layout graphics with `Phylo.draw`, and the examples are specifically there to show these modes. In #1247 `draw_graphviz` was deprecated before any replacement was implemented. I can see that it's not strictly necessary to have other than horizontal cladograms, but we're talking about these specific parts of the tutorial now. Going with #1247 I'm not interested in fixing the pydot issue.

**Comment by rwst** [comment]
Hold it. I can perhaps rewrite them using networkx, not sure how close I can get.

**Comment by rwst** [comment]
No, networkx is not suited for this.

**Comment by peterjc** [comment]
Closed via #1971
